### PR TITLE
chore: rename all enum cases to PascalCase instead of SHOUT_CASE

### LINKED ISSUE #448: chore: rename all enum cases to use PascalCase
enum with cases that need renaming:

- `Psl\Str\Encoding`
- `Psl\Html\Encoding`
- `Psl\Hash\Algorithm`
- `Psl\File\LockType`
- `Psl\File\WriteMode`

unsure if there is more that need renaming.

### PR DESCRIPTION
**Pull request by azjezz**
closes #448 

<details>
<summary>
List of renamed cases
</summary>

```md
- Renamed `Encoding\Base64\Variant::Default` to `Encoding\Base64\Variant::Standard`
- Renamed `File\LockType::SHARED` to `File\LockType::Shared`
- Renamed `File\LockType::EXCLUSIVE` to `File\LockType::Exclusive`
- Renamed `File\WriteMode::OPEN_OR_CREATE` to `File\WriteMode::OpenOrCreate`
- Renamed `File\WriteMode::TRUNCATE` to `File\WriteMode::Truncate`
- Renamed `File\WriteMode::APPEND` to `File\WriteMode::Append`
- Renamed `File\WriteMode::MUST_CREATE` to `File\WriteMode::MustCreate`
- Renamed `Hash\Algorithm::MD2` to `Hash\Algorithm::Md2`
- Renamed `Hash\Algorithm::MD4` to `Hash\Algorithm::Md4`
- Renamed `Hash\Algorithm::MD5` to `Hash\Algorithm::Md5`
- Renamed `Hash\Algorithm::SHA1` to `Hash\Algorithm::Sha1`
- Renamed `Hash\Algorithm::SHA224` to `Hash\Algorithm::Sha224`
- Renamed `Hash\Algorithm::SHA256` to `Hash\Algorithm::Sha256`
- Renamed `Hash\Algorithm::SHA384` to `Hash\Algorithm::Sha384`
- Renamed `Hash\Algorithm::SHA512_224` to `Hash\Algorithm::Sha512224`
- Renamed `Hash\Algorithm::SHA512_256` to `Hash\Algorithm::Sha512256`
- Renamed `Hash\Algorithm::SHA512` to `Hash\Algorithm::Sha512`
- Renamed `Hash\Algorithm::SHA3_224` to `Hash\Algorithm::Sha3224`
- Renamed `Hash\Algorithm::SHA3_256` to `Hash\Algorithm::Sha3256`
- Renamed `Hash\Algorithm::SHA3_384` to `Hash\Algorithm::Sha3384`
- Renamed `Hash\Algorithm::SHA3_512` to `Hash\Algorithm::Sha3512`
- Renamed `Hash\Algorithm::RIPEMD_128` to `Hash\Algorithm::Ripemd128`
- Renamed `Hash\Algorithm::RIPEMD_160` to `Hash\Algorithm::Ripemd160`
- Renamed `Hash\Algorithm::RIPEMD_256` to `Hash\Algorithm::Ripemd256`
- Renamed `Hash\Algorithm::RIPEMD_320` to `Hash\Algorithm::Ripemd320`
- Renamed `Hash\Algorithm::WHIRLPOOL` to `Hash\Algorithm::Whirlpool`
- Renamed `Hash\Algorithm::TIGER128_3` to `Hash\Algorithm::Tiger1283`
- Renamed `Hash\Algorithm::TIGER160_3` to `Hash\Algorithm::Tiger1603`
- Renamed `Hash\Algorithm::TIGER192_3` to `Hash\Algorithm::Tiger1923`
- Renamed `Hash\Algorithm::TIGER128_4` to `Hash\Algorithm::Tiger1284`
- Renamed `Hash\Algorithm::TIGER160_4` to `Hash\Algorithm::Tiger1604`
- Renamed `Hash\Algorithm::TIGER192_4` to `Hash\Algorithm::Tiger1924`
- Renamed `Hash\Algorithm::SNEFRU` to `Hash\Algorithm::Snefru`
- Renamed `Hash\Algorithm::SNEFRU_256` to `Hash\Algorithm::Snefru256`
- Renamed `Hash\Algorithm::GOST` to `Hash\Algorithm::Gost`
- Renamed `Hash\Algorithm::GOST_CRYPTO` to `Hash\Algorithm::GostCrypto`
- Renamed `Hash\Algorithm::ADLER32` to `Hash\Algorithm::Adler32`
- Renamed `Hash\Algorithm::CRC32` to `Hash\Algorithm::Crc32`
- Renamed `Hash\Algorithm::CRC32B` to `Hash\Algorithm::Crc32B`
- Renamed `Hash\Algorithm::CRC32C` to `Hash\Algorithm::Crc32C`
- Renamed `Hash\Algorithm::FNV132` to `Hash\Algorithm::Fnv132`
- Renamed `Hash\Algorithm::FNV1A32` to `Hash\Algorithm::Fnv1A32`
- Renamed `Hash\Algorithm::FNV164` to `Hash\Algorithm::Fnv164`
- Renamed `Hash\Algorithm::FNV1A64` to `Hash\Algorithm::Fnv1A64`
- Renamed `Hash\Algorithm::JOAAT` to `Hash\Algorithm::Joaat`
- Renamed `Hash\Algorithm::MURMUR3A` to `Hash\Algorithm::Murmur3A`
- Renamed `Hash\Algorithm::MURMUR3C` to `Hash\Algorithm::Murmur3C`
- Renamed `Hash\Algorithm::MURMUR3F` to `Hash\Algorithm::Murmur3F`
- Renamed `Hash\Algorithm::XXH32` to `Hash\Algorithm::Xxh32`
- Renamed `Hash\Algorithm::XXH64` to `Hash\Algorithm::Xxh64`
- Renamed `Hash\Algorithm::XXH3` to `Hash\Algorithm::Xxh3`
- Renamed `Hash\Algorithm::XXH128` to `Hash\Algorithm::Xxh128`
- Renamed `Hash\Algorithm::HAVAL128_3` to `Hash\Algorithm::Haval1283`
- Renamed `Hash\Algorithm::HAVAL160_3` to `Hash\Algorithm::Haval1603`
- Renamed `Hash\Algorithm::HAVAL192_3` to `Hash\Algorithm::Haval1923`
- Renamed `Hash\Algorithm::HAVAL224_3` to `Hash\Algorithm::Haval2243`
- Renamed `Hash\Algorithm::HAVAL256_3` to `Hash\Algorithm::Haval2563`
- Renamed `Hash\Algorithm::HAVAL128_4` to `Hash\Algorithm::Haval1284`
- Renamed `Hash\Algorithm::HAVAL160_4` to `Hash\Algorithm::Haval1604`
- Renamed `Hash\Algorithm::HAVAL192_4` to `Hash\Algorithm::Haval1924`
- Renamed `Hash\Algorithm::HAVAL224_4` to `Hash\Algorithm::Haval2244`
- Renamed `Hash\Algorithm::HAVAL256_4` to `Hash\Algorithm::Haval2564`
- Renamed `Hash\Algorithm::HAVAL128_5` to `Hash\Algorithm::Haval1285`
- Renamed `Hash\Algorithm::HAVAL160_5` to `Hash\Algorithm::Haval1605`
- Renamed `Hash\Algorithm::HAVAL192_5` to `Hash\Algorithm::Haval1925`
- Renamed `Hash\Algorithm::HAVAL224_5` to `Hash\Algorithm::Haval2245`
- Renamed `Hash\Algorithm::HAVAL256_5` to `Hash\Algorithm::Haval2565`
- Renamed `Html\Encoding::UTF_8` to `Html\Encoding::Utf8`
- Renamed `Html\Encoding::ISO_8859_1` to `Html\Encoding::Iso88591`
- Renamed `Html\Encoding::ISO_8859_15` to `Html\Encoding::Iso885915`
- Renamed `Html\Encoding::ISO_8859_5` to `Html\Encoding::Iso88595`
- Renamed `Html\Encoding::CP_866` to `Html\Encoding::Cp866`
- Renamed `Html\Encoding::CP_1251` to `Html\Encoding::Cp1251`
- Renamed `Html\Encoding::CP_1252` to `Html\Encoding::Cp1252`
- Renamed `Html\Encoding::KOI8_R` to `Html\Encoding::Koi8R`
- Renamed `Html\Encoding::BIG5` to `Html\Encoding::Big5`
- Renamed `Html\Encoding::GB2312` to `Html\Encoding::Gb2312`
- Renamed `Html\Encoding::BIG5_HKSCS` to `Html\Encoding::Big5Hkscs`
- Renamed `Html\Encoding::SHIFT_JIS` to `Html\Encoding::ShiftJis`
- Renamed `Html\Encoding::EUC_JP` to `Html\Encoding::EucJp`
- Renamed `Html\Encoding::MAC_ROMAN` to `Html\Encoding::MacRoman`
- Renamed `Network\SocketScheme::TCP` to `Network\SocketScheme::Tcp`
- Renamed `Network\SocketScheme::UNIX` to `Network\SocketScheme::Unix`
- Renamed `Str\Encoding::BASE64` to `Str\Encoding::Base64`
- Renamed `Str\Encoding::UUENCODE` to `Str\Encoding::Uuencode`
- Renamed `Str\Encoding::HTML_ENTITIES` to `Str\Encoding::HtmlEntities`
- Renamed `Str\Encoding::QUOTED_PRINTABLE` to `Str\Encoding::QuotedPrintable`
- Renamed `Str\Encoding::ASCII_7BIT` to `Str\Encoding::Ascii7bit`
- Renamed `Str\Encoding::ASCII_8BIT` to `Str\Encoding::Ascii8bit`
- Renamed `Str\Encoding::UCS_4` to `Str\Encoding::Ucs4`
- Renamed `Str\Encoding::UCS_4BE` to `Str\Encoding::Ucs4be`
- Renamed `Str\Encoding::UCS_4LE` to `Str\Encoding::Ucs4le`
- Renamed `Str\Encoding::UCS_2` to `Str\Encoding::Ucs2`
- Renamed `Str\Encoding::UCS_2BE` to `Str\Encoding::Ucs2be`
- Renamed `Str\Encoding::UCS_2LE` to `Str\Encoding::Ucs2le`
- Renamed `Str\Encoding::UTF_32` to `Str\Encoding::Utf32`
- Renamed `Str\Encoding::UTF_32BE` to `Str\Encoding::Utf32be`
- Renamed `Str\Encoding::UTF_32LE` to `Str\Encoding::Utf32le`
- Renamed `Str\Encoding::UTF_16` to `Str\Encoding::Utf16`
- Renamed `Str\Encoding::UTF_16BE` to `Str\Encoding::Utf16be`
- Renamed `Str\Encoding::UTF_16LE` to `Str\Encoding::Utf16le`
- Renamed `Str\Encoding::UTF_8` to `Str\Encoding::Utf8`
- Renamed `Str\Encoding::UTF_7` to `Str\Encoding::Utf7`
- Renamed `Str\Encoding::UTF7_IMAP` to `Str\Encoding::Utf7Imap`
- Renamed `Str\Encoding::ASCII` to `Str\Encoding::Ascii`
- Renamed `Str\Encoding::EUC_JP` to `Str\Encoding::EucJp`
- Renamed `Str\Encoding::SJIS` to `Str\Encoding::Sjis`
- Renamed `Str\Encoding::EUCJP_WIN` to `Str\Encoding::EucjpWin`
- Renamed `Str\Encoding::EUC_JP_2004` to `Str\Encoding::EucJp2004`
- Renamed `Str\Encoding::SJIS_MOBILE_DOCOMO` to `Str\Encoding::SjisMobileDocomo`
- Renamed `Str\Encoding::SJIS_MOBILE_KDDI` to `Str\Encoding::SjisMobileKddi`
- Renamed `Str\Encoding::SJIS_MOBILE_SOFTBANK` to `Str\Encoding::SjisMobileSoftbank`
- Renamed `Str\Encoding::SJIS_MAC` to `Str\Encoding::SjisMac`
- Renamed `Str\Encoding::SJIS_2004` to `Str\Encoding::Sjis2004`
- Renamed `Str\Encoding::UTF_8_MOBILE_DOCOMO` to `Str\Encoding::Utf8MobileDocomo`
- Renamed `Str\Encoding::UTF_8_MOBILE_KDDI_A` to `Str\Encoding::Utf8MobileKddiA`
- Renamed `Str\Encoding::UTF_8_MOBILE_KDDI_B` to `Str\Encoding::Utf8MobileKddiB`
- Renamed `Str\Encoding::UTF_8_MOBILE_SOFTBANK` to `Str\Encoding::Utf8MobileSoftbank`
- Renamed `Str\Encoding::CP932` to `Str\Encoding::Cp932`
- Renamed `Str\Encoding::CP51932` to `Str\Encoding::Cp51932`
- Renamed `Str\Encoding::JIS` to `Str\Encoding::Jis`
- Renamed `Str\Encoding::ISO_2022_JP` to `Str\Encoding::Iso2022Jp`
- Renamed `Str\Encoding::ISO_2022_JP_MS` to `Str\Encoding::Iso2022JpMs`
- Renamed `Str\Encoding::GB18030` to `Str\Encoding::Gb18030`
- Renamed `Str\Encoding::WINDOWS_1252` to `Str\Encoding::Windows1252`
- Renamed `Str\Encoding::WINDOWS_1254` to `Str\Encoding::Windows1254`
- Renamed `Str\Encoding::ISO_8859_1` to `Str\Encoding::Iso88591`
- Renamed `Str\Encoding::ISO_8859_2` to `Str\Encoding::Iso88592`
- Renamed `Str\Encoding::ISO_8859_3` to `Str\Encoding::Iso88593`
- Renamed `Str\Encoding::ISO_8859_4` to `Str\Encoding::Iso88594`
- Renamed `Str\Encoding::ISO_8859_5` to `Str\Encoding::Iso88595`
- Renamed `Str\Encoding::ISO_8859_6` to `Str\Encoding::Iso88596`
- Renamed `Str\Encoding::ISO_8859_7` to `Str\Encoding::Iso88597`
- Renamed `Str\Encoding::ISO_8859_8` to `Str\Encoding::Iso88598`
- Renamed `Str\Encoding::ISO_8859_9` to `Str\Encoding::Iso88599`
- Renamed `Str\Encoding::ISO_8859_10` to `Str\Encoding::Iso885910`
- Renamed `Str\Encoding::ISO_8859_13` to `Str\Encoding::Iso885913`
- Renamed `Str\Encoding::ISO_8859_14` to `Str\Encoding::Iso885914`
- Renamed `Str\Encoding::ISO_8859_15` to `Str\Encoding::Iso885915`
- Renamed `Str\Encoding::ISO_8859_16` to `Str\Encoding::Iso885916`
- Renamed `Str\Encoding::EUC_CN` to `Str\Encoding::EucCn`
- Renamed `Str\Encoding::CP936` to `Str\Encoding::Cp936`
- Renamed `Str\Encoding::HZ` to `Str\Encoding::Hz`
- Renamed `Str\Encoding::EUC_TW` to `Str\Encoding::EucTw`
- Renamed `Str\Encoding::BIG_5` to `Str\Encoding::Big5`
- Renamed `Str\Encoding::CP950` to `Str\Encoding::Cp950`
- Renamed `Str\Encoding::EUC_KR` to `Str\Encoding::EucKr`
- Renamed `Str\Encoding::UHC` to `Str\Encoding::Uhc`
- Renamed `Str\Encoding::ISO_2022_KR` to `Str\Encoding::Iso2022Kr`
- Renamed `Str\Encoding::WINDOWS_1251` to `Str\Encoding::Windows1251`
- Renamed `Str\Encoding::CP866` to `Str\Encoding::Cp866`
- Renamed `Str\Encoding::KOI8_R` to `Str\Encoding::Koi8R`
- Renamed `Str\Encoding::KOI8_U` to `Str\Encoding::Koi8U`
- Renamed `Str\Encoding::ARMSCII_8` to `Str\Encoding::Armscii8`
- Renamed `Str\Encoding::CP850` to `Str\Encoding::Cp850`
- Renamed `Str\Encoding::ISO_2022_JP_2004` to `Str\Encoding::Iso2022Jp2004`
- Renamed `Str\Encoding::ISO_2022_JP_MOBILE_KDDI` to `Str\Encoding::Iso2022JpMobileKddi`
- Renamed `Str\Encoding::CP50220` to `Str\Encoding::Cp50220`
- Renamed `Str\Encoding::CP50221` to `Str\Encoding::Cp50221`
- Renamed `Str\Encoding::CP50222` to `Str\Encoding::Cp50222`
- Renamed `Hash\Hmac\Algorithm::MD2` to `Hash\Hmac\Algorithm::Md2`
- Renamed `Hash\Hmac\Algorithm::MD4` to `Hash\Hmac\Algorithm::Md4`
- Renamed `Hash\Hmac\Algorithm::MD5` to `Hash\Hmac\Algorithm::Md5`
- Renamed `Hash\Hmac\Algorithm::SHA1` to `Hash\Hmac\Algorithm::Sha1`
- Renamed `Hash\Hmac\Algorithm::SHA224` to `Hash\Hmac\Algorithm::Sha224`
- Renamed `Hash\Hmac\Algorithm::SHA256` to `Hash\Hmac\Algorithm::Sha256`
- Renamed `Hash\Hmac\Algorithm::SHA384` to `Hash\Hmac\Algorithm::Sha384`
- Renamed `Hash\Hmac\Algorithm::SHA512_224` to `Hash\Hmac\Algorithm::Sha512224`
- Renamed `Hash\Hmac\Algorithm::SHA512_256` to `Hash\Hmac\Algorithm::Sha512256`
- Renamed `Hash\Hmac\Algorithm::SHA512` to `Hash\Hmac\Algorithm::Sha512`
- Renamed `Hash\Hmac\Algorithm::SHA3_224` to `Hash\Hmac\Algorithm::Sha3224`
- Renamed `Hash\Hmac\Algorithm::SHA3_256` to `Hash\Hmac\Algorithm::Sha3256`
- Renamed `Hash\Hmac\Algorithm::SHA3_384` to `Hash\Hmac\Algorithm::Sha3384`
- Renamed `Hash\Hmac\Algorithm::SHA3_512` to `Hash\Hmac\Algorithm::Sha3512`
- Renamed `Hash\Hmac\Algorithm::RIPEMD_128` to `Hash\Hmac\Algorithm::Ripemd128`
- Renamed `Hash\Hmac\Algorithm::RIPEMD_160` to `Hash\Hmac\Algorithm::Ripemd160`
- Renamed `Hash\Hmac\Algorithm::RIPEMD_256` to `Hash\Hmac\Algorithm::Ripemd256`
- Renamed `Hash\Hmac\Algorithm::RIPEMD_320` to `Hash\Hmac\Algorithm::Ripemd320`
- Renamed `Hash\Hmac\Algorithm::WHIRLPOOL` to `Hash\Hmac\Algorithm::Whirlpool`
- Renamed `Hash\Hmac\Algorithm::TIGER128_3` to `Hash\Hmac\Algorithm::Tiger1283`
- Renamed `Hash\Hmac\Algorithm::TIGER160_3` to `Hash\Hmac\Algorithm::Tiger1603`
- Renamed `Hash\Hmac\Algorithm::TIGER192_3` to `Hash\Hmac\Algorithm::Tiger1923`
- Renamed `Hash\Hmac\Algorithm::TIGER128_4` to `Hash\Hmac\Algorithm::Tiger1284`
- Renamed `Hash\Hmac\Algorithm::TIGER160_4` to `Hash\Hmac\Algorithm::Tiger1604`
- Renamed `Hash\Hmac\Algorithm::TIGER192_4` to `Hash\Hmac\Algorithm::Tiger1924`
- Renamed `Hash\Hmac\Algorithm::SNEFRU` to `Hash\Hmac\Algorithm::Snefru`
- Renamed `Hash\Hmac\Algorithm::SNEFRU_256` to `Hash\Hmac\Algorithm::Snefru256`
- Renamed `Hash\Hmac\Algorithm::GOST` to `Hash\Hmac\Algorithm::Gost`
- Renamed `Hash\Hmac\Algorithm::GOST_CRYPTO` to `Hash\Hmac\Algorithm::GostCrypto`
- Renamed `Hash\Hmac\Algorithm::HAVAL128_3` to `Hash\Hmac\Algorithm::Haval1283`
- Renamed `Hash\Hmac\Algorithm::HAVAL160_3` to `Hash\Hmac\Algorithm::Haval1603`
- Renamed `Hash\Hmac\Algorithm::HAVAL192_3` to `Hash\Hmac\Algorithm::Haval1923`
- Renamed `Hash\Hmac\Algorithm::HAVAL224_3` to `Hash\Hmac\Algorithm::Haval2243`
- Renamed `Hash\Hmac\Algorithm::HAVAL256_3` to `Hash\Hmac\Algorithm::Haval2563`
- Renamed `Hash\Hmac\Algorithm::HAVAL128_4` to `Hash\Hmac\Algorithm::Haval1284`
- Renamed `Hash\Hmac\Algorithm::HAVAL160_4` to `Hash\Hmac\Algorithm::Haval1605`
- Renamed `Hash\Hmac\Algorithm::HAVAL192_3` to `Hash\Hmac\Algorithm::Haval1923`
- Renamed `Hash\Hmac\Algorithm::HAVAL224_3` to `Hash\Hmac\Algorithm::Haval2243`
- Renamed `Hash\Hmac\Algorithm::HAVAL256_3` to `Hash\Hmac\Algorithm::Haval2563`
- Renamed `Hash\Hmac\Algorithm::HAVAL128_4` to `Hash\Hmac\Algorithm::Haval1284`
- Renamed `Hash\Hmac\Algorithm::HAVAL160_4` to `Hash\Hmac\Algorithm::Haval1604`
- Renamed `Hash\Hmac\Algorithm::HAVAL192_4` to `Hash\Hmac\Algorithm::Haval1924`
- Renamed `Hash\Hmac\Algorithm::HAVAL224_4` to `Hash\Hmac\Algorithm::Haval2244`
- Renamed `Hash\Hmac\Algorithm::HAVAL256_4` to `Hash\Hmac\Algorithm::Haval2564`
- Renamed `Hash\Hmac\Algorithm::HAVAL128_5` to `Hash\Hmac\Algorithm::Haval1285`
- Renamed `Hash\Hmac\Algorithm::HAVAL160_5` to `Hash\Hmac\Algorithm::Haval1605`
- Renamed `Hash\Hmac\Algorithm::HAVAL192_5` to `Hash\Hmac\Algorithm::Haval1925`
- Renamed `Hash\Hmac\Algorithm::HAVAL224_5` to `Hash\Hmac\Algorithm::Haval2245`
- Renamed `Hash\Hmac\Algorithm::HAVAL256_5` to `Hash\Hmac\Algorithm::Haval2565`
```

</details>

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,9 +6,8 @@
 
 # Components
 
-* [Psl\Async](../src/Psl/Async/README.md)
-* [Psl\Default](../src/Psl/Default/README.md)
```
Comment (veewee) [inline docs/README.md:10]: this removed default which is now in next?

**Review by veewee — APPROVED** [review]
It's gonna be a bit of an annoying BC break for downstream usage though.
Just got to ask : Is it actually worth it? :)

File: docs/README.md (hunk 1 above)
Comment (azjezz) [reply docs/README.md:10]: nice catch!

**Comment by azjezz** [comment]
> It's gonna be a bit of an annoying BC break for downstream usage though.
> Just got to ask : Is it actually worth it? :)

I'm yes, it is worth it, for consistency sake.

I do not assume that BC breaks are going to be a huge problem downstream, and its a major release, so BC breaks are expected.

**Comment by lexidor** [comment]
Is there no way to leave an alias in place with the old name? This change make it incredibly tedious for libraries to support. Ideally, libraries that depend on this library could advertise `^2.y.z || ^3` by putting in a lot of effort. Chasm changes, where writing version agnostic code is very hard, have a lot of impact on the module graph. This is especially true in a dependency which is a dependency of other dependencies.

**Comment by azjezz** [comment]
> Is there no way to leave an alias in place with the old name

quity tricky, and i don't want to include hacks in Psl for compatibilty.

I don't recommand doing `2.x || 3.x`, as major releases in Psl tend to have way too many changes ( e.g from 1 to 2 we basically re-wrote everything that is related to IO, changed multiple components completely, deleted components, and add new ones: https://github.com/azjezz/psl/compare/1.9.2...2.0.0 ).

**Comment by faizanakram99** [comment]
> Is there no way to leave an alias in place with the old name? This change make it incredibly tedious for libraries to support. Ideally, libraries that depend on this library could advertise `^2.y.z || ^3` by putting in a lot of effort. Chasm changes, where writing version agnostic code is very hard, have a lot of impact on the module graph. This is especially true in a dependency which is a dependency of other dependencies.

maybe create a rector rule for upgrading SHOUT_CASE to PascalCase?

**Comment by azjezz** [comment]
> maybe create a rector rule for upgrading SHOUT_CASE to PascalCase?

That is indeed a good idea 👍 I'm working on many things related to 3.x right now, and we might want to create rules for rector to upgrade.

personally i don't think i have the time to do that, but if someone is willing to create the rules, we can have them in the github.com/php-standard-library org.

**Pull request merged**